backend/scripts/dev/docker-compose.sh

```
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+RELATIVE_DIR=`dirname "$0"`
+
+cd $RELATIVE_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)